tests/base/src/test/java/org/wildfly/security/ldap/PrincipalMappingSuiteChild.java

```diff
@@ -70,36 +74,12 @@ public void testDnToSimple() throws RealmUnavailableException {
 
     @Test
     public void testSimpleToSimpleValidate() throws RealmUnavailableException {
-        SecurityRealm realm = LdapSecurityRealmBuilder.builder()
-                .setDirContextSupplier(LdapTestSuite.dirContextFactory.create())
-                .identityMapping()
-                    .setSearchDn("dc=elytron,dc=wildfly,dc=org")
-                    .setRdnIdentifier("uid")
-                    .build()
-                .build();
-
-        RealmIdentity identity = realm.getRealmIdentity(new NamePrincipal("PlainUser"));
-        assertTrue("Exists", identity.exists());
-
-        identity = realm.getRealmIdentity(new NamePrincipal("nobody"));
-        assertFalse("Exists", identity.exists());
+        testSimpleCommon();
     }
 
     @Test
     public void testSimpleToSimpleReload() throws RealmUnavailableException {
-        SecurityRealm realm = LdapSecurityRealmBuilder.builder()
-                .setDirContextSupplier(LdapTestSuite.dirContextFactory.create())
-                .identityMapping()
-                    .setSearchDn("dc=elytron,dc=wildfly,dc=org")
-                    .setRdnIdentifier("uid")
-                    .build()
-                .build();
-
-        RealmIdentity identity = realm.getRealmIdentity(new NamePrincipal("PlainUser"));
-        assertTrue("Exists", identity.exists());
-
-        identity = realm.getRealmIdentity(new NamePrincipal("nobody"));
-        assertFalse("Exists", identity.exists());
+        testSimpleCommon();
```

Comment:
@map-b It appears that there is no difference between the test methods `testSimpleToSimpleReload` , `testSimpleToSimpleValidate` , and `testSimpleToDn` . 

Since that is the case, please remove the unneeded repeated tests and leave just 1 test named for example `testLdapRealmPrincipalMapping`

Reply:
Done. Tell me if you see more redundant code. Thanks.